public/js/common.js

```diff
@@ -68,3 +68,113 @@ function failSnackbar(msg) {
 $(document).on('click', '#closeSnack', function() {
     $(this).parent().fadeOut();
 });
+
+
+/*
+These errors are only for UI use
+Error codes and their corresponding message. Error codes are under different
+categories:
+1000 -> system
+2000 -> user
+3000 -> question
+4000 -> class
+5000 -> analytics
+6000 -> import/export
+7000 -> settings
+*/
+
+const errors = Object.freeze({
+    //1000 system
+    1000: 'Invalid Request',
+    1001: 'Missing Requirement',
+    1002: 'Permission Denied',
+    1003: 'Confirm password doesn\'t match',
+    1004: 'Database connection failed',
+    1005: 'Database Error',
```

Comment:
can we get more details on what this error is/where it happens at?

Reply:
This error happens when we try to validate the password in the backend, since this is the UI, we should not be displaying to the user that we could not validate the password, since the user wouldn't care about that